soql-standalone-parser/src/main/scala/com/socrata/soql/ast/Expression.scala

```diff
@@ -293,9 +294,10 @@ case class FunctionCall(functionName: FunctionName, parameters: Seq[Expression],
         }
         (whens ++ otherwise).encloseNesting(d"CASE" flatAlt d"CASE ", d"", d"END" flatAlt d" END")
       case other =>
+        val filterd = filter.fold(Doc.empty) { f => d" FILTER(WHERE ${f.doc})" }
         val close = window.fold(Doc.empty) { w => d" " ++ w.doc }
         if(parameters.lengthCompare(1) == 0) {
-          d"${other.toString}(${parameters(0).doc})$close"
+          d"${other.toString}(${parameters(0).doc})$filterd$close"
```

Comment:
And this might be better as something along the lines of

```
(Seq(Doc(other.toString) ++ d"(" ++ parameters(0).doc ++ d") ++ filter ++ close)).vsep.hanging(2)
```
..in which case you'll want to get rid of the leading spaces on `filter` and `close`.